db/fixtures/ae_datastore/ManageIQ/Cloud/VM/Retirement/StateMachines/Methods.class/__methods__/microsoft_pre_retirement.rb

```diff
@@ -0,0 +1,10 @@
+#
+# Description: This method stops the Azure Instance
+#
+
+vm = $evm.root['vm']
+if vm && vm.power_state == 'on'
+  ems = vm.ext_management_system
+  $evm.log('info', "Stopping Azure Instance <#{vm.name}> in EMS <#{ems ? ems.name : nil}>")
```

Comment:
I know it is cut-and-paste from the other methods but  `ems ? ems.name : nil` can be reduced to `ems.try(:name)`